tools/lkl/lib/hijack/hijack.c

```diff
@@ -291,7 +304,25 @@ int select(int nfds, fd_set *r, fd_set *w, fd_set *e, struct timeval *t)
 	return lkl_call(__lkl__NR_select, 5, nfds, r, w, e, t);
 }
 
-HOOK_CALL_USE_HOST_BEFORE_START(epoll_create);
+HOST_CALL(epoll_create);
+int epoll_create(int flags)
```

Comment:
Hi Hajime

You were perfectly right. In the end it worked. Sorry for my misunderstanding.
I submitted the changes to https://github.com/lkl/linux.

Best regards,
  Octavian Soldea